compile.py

```diff
@@ -16,9 +25,24 @@
                         'dtrace_probe.cc',
                         'dtrace_argument.cc'
                     ],
-                    'include_dirs': [
-                      'libusdt',
-                      '<!(node -e "require(\'nan\')")',
+                    'conditions': [
+                        ['OS=="mac" or OS=="solaris"',
+                            { 'include_dirs': [
+                                 'libusdt',
+                                 '<!(node -e "require(\'nan\')")',
+                              ]
+                            }
+                        ],
+                        ['OS=="freebsd"',
+                            { 'include_dirs': [
+                                  '/usr/src/cddl/compat/opensolaris/',
```

Comment:
I don't know enough about using and building on BSD.  Is it expected that components hardcode full paths like this?